user-metrics/metrics-service-datadog/metrics-service.yml.template

```diff
@@ -1,7 +1,7 @@
 spec:
   container:
     - name: mdservice
-      image: ${SPCS_IMAGE_REPO}/mdservice:v0.0.1
+      image: SPCS_IMAGE_REPO/mdservice:VERSION
```

Comment:
is it a right change?

Reply:
yes verified.
The reason this has to be done is customers don't have gettext by default on their machine
using sed to replace it